src/main/replayBrowser/db.sql

```diff
@@ -0,0 +1,131 @@
+CREATE TABLE IF NOT EXISTS replays (
+    id       INTEGER PRIMARY KEY AUTOINCREMENT,
+    fullPath TEXT NOT,
+    name     TEXT NOT NULL,
+    folder   TEXT NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS replays_full_path_idx ON replays(fullPath)
+CREATE INDEX IF NOT EXISTS replays_folder_idx ON replays(folder)
+
+CREATE TABLE IF NOT EXISTS replay_data (
+    gameId             INTEGER PRIMARY KEY,
+    startTime          TEXT,
+    lastFrame          INTEGER,
+    slpVersion         TEXT,
+    isPal              BOOLEAN,
+    stageId            INTEGER,
+    scene              INTEGER,
+    playedOn           TEXT,
+    gameComplete       BOOLEAN,
+    playableFrameCount INTEGER,
+    gameMode           INTEGER
+    FOREIGN KEY (gameId) REFERENCES replays(gameId) ON DELETE CASCADE
+);
+
+CREATE INDEX IF NOT EXISTS replay_data_start_time_path_idx ON replay_data(startTime)
+
+CREATE TABLE IF NOT EXISTS player (
+    gameId           INTEGER,
+    playerIndex      INTEGER,
+    nametag          TEXT,
+    playerCode       TEXT,
+    port             INTEGER,
+    characterId      INTEGER,
+    characterColor   Integer,
+    startStocks      Integer,
```

Comment:
i feel like there isn't much point in storing `startStocks`? `endPercent` could be good for calculating tie-breaks when a game reaches time though.